package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "aas-frontend",
+  "name": "argus-frontend",
```

```suggestion
  "name": "Argus-frontend",
```

Also, I assume that the corresponding name is automatically changed in `package-lock.json` if this change is made..?